src/Compiler/Checking/MethodOverrides.fs

```diff
@@ -831,7 +831,7 @@ module DispatchSlotChecking =
                     let allCorrect = CheckDispatchSlotsAreImplemented (denv, infoReader, m, nenv, sink, tcaug.tcaug_abstract, false, reqdTy, dispatchSlots, availPriorOverrides, overrides)
                     
                     // Tell the user to mark the thing abstract if it was missing implementations
-                    if not allCorrect && not tcaug.tcaug_abstract && not (isInterfaceTy g reqdTy) then 
```

Comment:
By checking for not (isInterfaceTy g reqdTy) we we also reporting on records and unions..

Reply:
If abstract / abstract member is added to a DU/Record, it keeps reporting the:
`error FS0912: This declaration element is not permitted in an augmentation`
error, right?

Reply:
Yes.